machineProblem.py

```diff
@@ -126,6 +126,12 @@ class FirstQuestion(tkinter.Frame):
     def __init__(self, parent, controller):
         tkinter.Frame.__init__(self, parent)
         
+        f = None
+        df = pd.read_csv('questionBank.csv')
+
+        print(df)
```

Comment:
this only prints the csv in the terminal. think of the main design, and you might consider commenting/deleting this out.